RatS/allocine/allocine_ratings_parser.py

```diff
@@ -0,0 +1,92 @@
+from bs4 import BeautifulSoup
+
+from RatS.base.base_ratings_parser import RatingsParser
+from RatS.allocine.allocine_site import Allocine
+
+
+class AllocineRatingsParser(RatingsParser):
+    def __init__(self, args):
+        super(AllocineRatingsParser, self).__init__(Allocine(args), args)
+
+    def _get_ratings_page(self, i):
+        return '{url}?page={page_number}'.format(url=self.site.MY_RATINGS_URL, page_number=i)
+
+    # Override to avoid redundant operations
+    def _retrieve_pages_count_and_movies_count(self, movie_ratings_page):
+        pages_count = self._get_pages_count(movie_ratings_page)
+        # Only add the number of movie on the last page
+        self.site.browser.get(self._get_ratings_page(pages_count))
+        last_page = BeautifulSoup(self.site.browser.page_source, 'html.parser')
+        last_page_count = len(last_page.find_all('div', class_='card'))
+        self.movies_count = (pages_count - 1) * 36 + last_page_count
+        return pages_count
+
+    @staticmethod
+    def _get_pages_count(movie_ratings_page):
+        pages_count = 1
+        pagination_holder = movie_ratings_page.find('div', class_='pagination-item-holder')
+        if pagination_holder:
+            pages_count = max(list(map(lambda tag: int(tag.text), pagination_holder.find_all('a', text=True))))
+
+        return pages_count
+
+    @staticmethod
+    def _get_movie_tiles(movie_listing_page):
+        return movie_listing_page.find_all('div', class_='card')
+
+    @staticmethod
+    def _get_movie_title(movie_tile):
+        # Original movie title is found later on the movie details page
```

Comment:
Suggestion for improving the clarification comment:
`At this point we only see the French movie title. Therefore, the original movie title is parsed later on the movie details page using _get_movie_original_title`